util/log/log.go

```diff
@@ -17,7 +17,52 @@
 
 package log
 
-import "golang.org/x/net/context"
+import (
+	"strings"
+
+	"golang.org/x/net/context"
+)
+
+// Level Enum for log severity levels.
+type Level int
+
+// TODO(Bram): combine this with clog.severity, and rename severity.
+
+// These constants identify the log levels in order of increasing severity.
+const (
+	INFO Level = iota
+	WARNING
+	ERROR
+	FATAL
+)
+
+var levels = [...]string{
+	"INFO",
```

Comment:
this should use stringer

I think you should merge this with clog.severity here so we can avoid reviewing both